### DISCUSSION TITLE
Command to recalculate with config parametter

### DISCUSSION THREAD

**Discussion by SwartZCoding** [BG-Software-LLC/SuperiorSkyblock2#1117] [Pending Feature Requests]
Hello, I made the mistake of changing the schematic-offset parameter in the middle of my version.

I think it would be a good idea to create an administrator command that would allow to recalculate the island points according to the modification of this parameter.

Or, to create a new command which "forces" the recalculation of all the islands levels with respect to the current configuration.

Thanks.

**Comment by OmerBenGera** [comment]
You can revert the change and use /is admin setbonus to fix the bonus for islands that were created in that period of time

**Comment by Lemvs** [comment]
@OmerBenGera you can close it.
